platform/bukkit/api/pom.xml

```diff
@@ -9,7 +9,7 @@
     <parent>
         <artifactId>treasury-parent</artifactId>
         <groupId>me.lokka30</groupId>
-        <version>1.0.1-SNAPSHOT</version>
+        <version>1.1-SNAPSHOT</version>
```

Comment:
`1.1.0-SNAPSHOT`

Reply:
current 1.0.1-SNAPSHOT version is fine. yannick should revert that change